$DragonFly: src/gnu/usr.bin/gdb/libgdb/patches/Attic/i386fbsd-nat.c.patch,v 1.2 2007/01/08 03:33:32 dillon Exp $

Index: i386fbsd-nat.c
===================================================================
RCS file: /cvs/src/contrib/gdb-6.2.1/gdb/i386fbsd-nat.c,v
retrieving revision 1.1
diff -u -r1.1 i386fbsd-nat.c
--- i386fbsd-nat.c	24 Oct 2004 19:56:25 -0000	1.1
+++ i386fbsd-nat.c	8 Jan 2007 01:11:49 -0000
@@ -107,7 +107,6 @@
   regcache_raw_supply (regcache, I386_ESP_REGNUM, &pcb->pcb_esp);
   regcache_raw_supply (regcache, I386_EBX_REGNUM, &pcb->pcb_ebx);
   regcache_raw_supply (regcache, I386_EIP_REGNUM, &pcb->pcb_eip);
-  regcache_raw_supply (regcache, I386_GS_REGNUM, &pcb->pcb_gs);
 
   return 1;
 }
@@ -141,7 +140,4 @@
       }
   }
 #endif
-
-  /* Support debugging kernel virtual memory images.  */
-  bsd_kvm_add_target (i386fbsd_supply_pcb);
 }
